screening-schema.json

```diff
@@ -323,7 +323,7 @@
             },
             "ensurePrivacySecurity": {
               "type": "string",
-              "description": "If yes - does the project ensure the privacy and security of this data and has it taken steps to prevent adverse impacts resulting from its collection, storage and distribution.",
+              "description": "If yes - does the project ensure the privacy and security of this data and has it taken steps to prevent adverse impacts resulting from its collection, storage and distribution. If yes - does this project share this data with third parties? Please describe the circumstances with which this project shares data with third parties. Please add links as relevant.",
```

Comment:
I believe some of these questions already exist in the schema for example:
              

- `Line 305`: "description": "If yes - does this project share this data with third parties?",